src/schema/v2/home/home_page_my_collection_onboarding_module.ts

```diff
@@ -0,0 +1,52 @@
+import { GraphQLBoolean, GraphQLObjectType } from "graphql"
+import { ResolverContext } from "types/graphql"
+
+const MyCollectionOnboardingModuleType = new GraphQLObjectType<
+  any,
+  ResolverContext
+>({
+  name: "HomePageMyCollectionOnboardingModule",
+  fields: {
+    showMyCollectionCard: {
+      type: GraphQLBoolean,
+      resolve: async (_root, _options, { collectionLoader, meLoader }) => {
+        if (!collectionLoader || !meLoader) {
+          return false
+        }
+
+        return meLoader().then((me) => {
+          return collectionLoader("my-collection", {
+            user_id: me.id,
+            private: true,
+          }).then((res) => {
+            return res.artworks_count > 3
+          })
+        })
+      },
+    },
+    showSWACard: {
+      type: GraphQLBoolean,
+      resolve: async (_root, _options, { collectionLoader, meLoader }) => {
+        if (!collectionLoader || !meLoader) {
+          return false
+        }
+
+        return meLoader().then((me) => {
+          return collectionLoader("my-collection", {
+            user_id: me.id,
+            private: true,
+          }).then((res) => {
+            return res.artworks_count > 3
+          })
+        })
+      },
+    },
+  },
```

Comment:
**issue(blocker):** I think you copied the code here without adjusting it, for now we need to return true always according to product specs

Reply:
Hm, no I didn't
Will revisit this tomorrow

Reply:
True, but I believe that's the logic only for my collection onboarding card. The swa card is supposed to be always visible I believe